README_MACOSX_NOTES.md

```diff
@@ -243,6 +207,10 @@ rake db:drop
 mysql -u root -p < db/initialize.sql
 ```
 
+NOTE: 2024-12-27 @JoeCohen: For me (but not others) `rake db:drop` threw an error
+because `config/database.yml` was missing.
+Please contact us if that is the case.
```

Comment:
@mo-nathan:
I didn't add more about this because `config/database.yml` is outside version control for confidentiality reasons. 
I also didn't add what we later did in mysql because I don't know if would publish confidential info.

Reply:
@JoeCohen We should add a `config/database.yml-template` file to address this, but that can happen in another PR.